server/views/pages/visitors.html

```diff
@@ -0,0 +1,124 @@
+{% from "../components/page-navigation/macro.njk" import pageNavigation %}
+{% from "../components/user-details/macro.njk" import userDetails %}
+{% from "../components/feedback-widget/macro.njk" import hubFeedbackWidget %}
+
+
+{% extends "../components/template.njk" %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block head %}
+  <link href="/public/stylesheets/application.css" rel="stylesheet"/>
+{% endblock %}
+
+{% block pageNavigation %}
+  {{ pageNavigation({ title: title }) }}
+{% endblock %}
+
+{% block topBar %}
+  {{ topBar({
+    establishmentDisplayName: establishmentDisplayName,
+    showBar: false,
+    userName: signedInUser,
+    returnUrl: returnUrl
+  }) }}
+{% endblock %}
+
+{% block userDetails %}
+  {{ userDetails({
+    detailsType: detailsType,
+    userName: signedInUser
+  }) }}
+{% endblock %}
+
+{% block header %}
+  <div class="govuk-width-container govuk-body">
+    <h1>{{ title }}</h1>
+  </div>
+{% endblock %}
+
+{% block homeNavigation %}{% endblock %}
+{% block search %}{% endblock %}
+
+{% block main %}
+  <div class="govuk-body">
+    {% if not signedInUser %}
+      <div class= "govuk-width-container">
+        <h2 class="govuk-heading-m">You are signed out</h2>
+        <p class="govuk-!-font-size-24" data-test="signin-prompt">
+          <a href="/auth/sign-in?returnUrl={{ returnUrl }}" class="govuk-link">Sign in</a> to see your personal information.
+          </p>
+      </div>
+    {% else %}
+      <div class= "govuk-width-container  govuk-!-padding-bottom-9">
+        {% set endVisitor = currentVisitorsPage*maxVisitorsPerPage %}
+        {% set startVisitor = endVisitor-maxVisitorsPerPage %}
+        {% set numberOfPages = (approvedVisitors.length/maxVisitorsPerPage) | round(0, "ceil") %}
+        {% if endVisitor > approvedVisitors.length %}
+          {% set endVisitor = approvedVisitors.length %}
+        {% endif %}
+        <dl class="govuk-summary-list">
+          {% for i in range(startVisitor, endVisitor) %}
+            <div class="govuk-summary-list__row">
+              <dd class="govuk-summary-list__value">
```

Comment:
think we'll need to add the attributes here to obfuscate the details from hotjar. 

`data-hj-suppress`

Reply:
Good catch